src/index.html

```diff
@@ -6,11 +6,17 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
you pasted it twice, you can remove one of those

Reply:
I saw it and I did that. I don't know where is my latest commit...
Tell me what should I do? Should I close PR and send another?

Reply:
just update the PR on develop branch and push it here, you can also remove `header-height` from variables - we're using it only once, so there is no point in creating variable for that 😉